modules/upgrade/pages/proxy-x.adoc

```diff
@@ -26,180 +19,100 @@ The distribution must be based on {sle}{nbsp}15 SP1, because {productname}{nbsp}
-        <product>SUSE Manager Proxy 4.0</product>
-        <product_dir>/</product_dir>
-      </listentry>
-      <listentry>
-        <ask_on_error config:type="boolean">true</ask_on_error>
-        <media_url>http://$redhat_management_server/ks/dist/child/sle-module-server-applications15-sp1-updates-x86_64-proxy-4.0/proxy4</media_url>
-        <name>sle-module-server-applications15-sp1-updates-x86_64-proxy-4.0</name>
-        <product>SUSE Manager Proxy 4.0</product>
-        <product_dir>/</product_dir>
-      </listentry>
-      <listentry>
-        <ask_on_error config:type="boolean">true</ask_on_error>
-        <media_url>http://$redhat_management_server/ks/dist/child/sle-module-suse-manager-proxy-4.0-pool-x86_64/proxy4</media_url>
-        <name>sle-module-suse-manager-proxy-4.0-pool-x86_64</name>
-        <product>SUSE Manager Proxy 4.0</product>
-        <product_dir>/</product_dir>
-      </listentry>
-      <listentry>
-        <ask_on_error config:type="boolean">true</ask_on_error>
-        <media_url>http://$redhat_management_server/ks/dist/child/sle-module-suse-manager-proxy-4.0-updates-x86_64/proxy4</media_url>
-        <name>sle-module-suse-manager-proxy-4.0-updates-x86_64</name>
-        <product>SUSE Manager Proxy 4.0</product>
-        <product_dir>/</product_dir>
-      </listentry>
-     </add_on_products>
-  </add-on>
-  <general>
-      $SNIPPET('spacewalk/sles_no_signature_checks')
-    <mode><confirm config:type="boolean">false</confirm></mode>
-  </general>
-  <upgrade>
-    <only_installed_packages config:type="boolean">false</only_installed_packages>
-    <stop_on_solver_conflict config:type="boolean">true</stop_on_solver_conflict>
-  </upgrade>
-  <backup>
-    <sysconfig config:type="boolean">true</sysconfig>
-    <modified config:type="boolean">true</modified>
-    <remove_old config:type="boolean">false</remove_old>
-  </backup>
-  <networking>
-    <keep_install_network config:type="boolean">true</keep_install_network>
-    <start_immediately config:type="boolean">true</start_immediately>
-  </networking>
-  <software>
-    <image/>
-    <install_recommended config:type="boolean">true</install_recommended>
-    <instsource/>
-    <patterns config:type="list">
-      <pattern>suma_proxy</pattern>
-    </patterns>
-    <products config:type="list">
-      <listentry>SUSE-Manager-Proxy</listentry>
-    </products>
-  </software>
-  <scripts>
-    <chroot-scripts config:type="list">
-      <script>
-        <filename>cleanup.sh</filename>
-        <chrooted config:type="boolean">true</chrooted>
-        <interpreter>shell</interpreter>
-        <source>
-        <![CDATA[
-#!/bin/sh
-         sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES wsgi
-         if [ -e /etc/rhn/rhn.conf.rpmsave ]; then
-             cp /etc/rhn/rhn.conf.rpmsave /etc/rhn/rhn.conf
-         fi
-         zypper rr --all
-         ]]>
-        </source>
-      </script>
-    </chroot-scripts>
-  </scripts>
-</profile>
```

Comment:
Please do not remove this example. This example is special for SUSE Manager Proxy.

Reply:
Ok, it seems you created a section for example profiles. This is good, but please reference the proxy profile here.

Reply:
L69:

`For example scripts that you can use as a starting point, see xref:upgrade:upgrade-script-example.adoc[].`